pages/configuration/logs.mdx

```diff
@@ -71,6 +71,31 @@ If you installed Memgraph using Docker:
     cat /var/log/memgraph/memgraph_2022-03-04.log
     ```
 
+    #### Filtering unnecessary log messages
+
+    Memgrpah uses several log files:
+    - General Logs: These are the default logs that track various levels such as `TRACE`, `DEBUG`, `INFO`, `WARNING`, `ERROR`, `CRITICAL`. 
+    - Audit Logs: These logs contain [records of all queries](/configuration/audit-log) executed on the database when enabled.
+    - Write-ahead Logs (WAL): These logs record [each database modification](/configuration/data-durability-and-backup) before it's written to the database, helping to reconstruct the database's most recent state.
+
+    When you use Memgraph's log files, you might find too many details that make it
+    hard to spot the important messages. To make analyzing the logs easier, you can
+    use a tool suach as `grep` to remove the messages you don't need. For instance,
+    if you want to ignore messages about internal Memgraph Lab work or too much information
+    on storage, indexes, rules, and triggers, you can use this grep command:

```

Comment:
This part is enough, I would just remove the above section. 

And not 'use log files', but 'inspect log files'